packages/material-ui/src/CssBaseline/CssBaseline.js

```diff
@@ -28,6 +28,12 @@ const useStyles = makeStyles(
           // Save printer ink.
           backgroundColor: theme.palette.common.white,
         },
+        '&::backdrop': {
+          backgroundColor: theme.palette.background.default,
+        },
+        '&::-ms-backdrop': {
```

Comment:
What's the use case for the ms prefix?

Reply:
Actually, you are right, I am not sure why did I add it in the first place now, I did this long time ago in one of my projects so I don't remember.